### ISSUE TITLE
Improved focus indicator contrast

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#9900]
Support making the focus indicator for the keyboard navigation black with white border (or similar) to ensure contrast on all color backgrounds.
